### PR TITLE
Drop support for old version of PHP and MW

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

File: ParserHooks.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	return 1;
 }
 
-define( 'ParserHooks_VERSION', '1.5.0' );
+define( 'ParserHooks_VERSION', '1.6.0 dev' );
```
Comment (kghbln) [inline ParserHooks.php:16]: The bunny is grumpy and rather wants to define in semver style 1.6.0-alpha.

55d26652b3143a728d23ba05e5d1b5da

File: README.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ This is a PHP library for MediaWiki extensions. It does not in itself add or enh
 * [PHP](http://php.net/) 5.3 or later (including PHP 7)
 * [MediaWiki](https://www.mediawiki.org/) 1.16 or later
 
+The latest versions (1.6+) requires more recent PHP and MediaWiki. If that does not work for you, just
+stick with version 1.5, which has no known defects and does not miss any real features. 
+
+* [PHP](http://php.net/) 5.5 or later (including PHP 7.x)
+* [MediaWiki](https://www.mediawiki.org/) 2.27 or later
```
Comment (kghbln) [inline README.md:24]: Time travel bunny does not find such a software

timebunny

Comment (JeroenDeDauw) [reply README.md:24]: :D Then time travel bunny went though time in the wrong direction!

**Comment by kghbln** [comment]
@JeroenDeDauw As soon as this is merged I could convert to PHP 5.4+ short array

**Pull request merged**